Makefile

```
@@ -377,10 +376,10 @@ go-deps: # download go dependencies
 docs-update: # update api docs
 	go run ./cmd/doc-gen/main.go
 
-e2e: ginkgo # make e2e tests
+fluentd-e2e: ginkgo # make e2e tests
 	tests/scripts/fluentd_e2e.sh
 
-helm-e2e: ginkgo # make helm e2e tests
+fluentd-helm-e2e: ginkgo # make helm e2e tests
 	tests/scripts/fluentd_helm_e2e.sh
```

Stick with the e2e task to keep it as closely aligned as the operator sdk to ease future migrations see https://v1-42-x.sdk.operatorframework.io/docs/upgrading-sdk-version/

```suggestion
.PHONY: e2e
e2e: e2e-fluentd ## run end to end tests

e2e-%: ginkgo 
	tests/scripts/$*_e2e.sh
```